modules/dxp/apps/commerce-avalara/commerce-avalara-tax-engine-fixed-web/src/main/java/com/liferay/commerce/avalara/tax/engine/fixed/configuration/CommerceTaxAvalaraTypeConfiguration.java

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ *
+ *
+ *
+ */
+
+package com.liferay.commerce.avalara.tax.engine.fixed.configuration;
```

Comment:
`Classes that are not exported should be in 'internal' package: ./modules/dxp/apps/commerce-avalara/commerce-avalara-tax-engine-fixed-web/src/main/java/com/liferay/commerce/avalara/tax/engine/fixed/configuration/CommerceTaxAvalaraTypeConfiguration.java (SourceCheck:JavaInternalPackageCheck)`

if this isn't planning to be exported it should be moved into `.internal.configuration;`.

Reply:
Got it, will update and fix. Thanks Danny.